### ISSUE TITLE
[Adaptive Cards-WeatherCompact.Json]: Narrator focus jumps degree F separately in scan mode by using up and down arrow key.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9247]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Cards Visualizer application.
3. Navigate to WeatherCompact.Json button and activate it.
4. Adaptive card will be appeared on right sight of the pane.
5. Navigate the adaptive card by using down arrow key in scan mode 
6. Now navigate to the degree F in scan mode by pressing down arrow key and verify the issue.

**Actual Result:**
Narrator focus jumps 46 and degree F separately in scan mode by using up and down arrow key and narrator announces degree F.

**Expected Result:**
Narrator focus should jump together on 46 and degree F and Narrator should announce 46 degree Fahrenheit.

**User Impact:**
It will be confusing & difficult for the users if Heading will not be defined inside the adaptive card.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/9a868c99-8b76-4ad3-950d-20a023f0f0dc

### Card JSON

```JSON
N/A
```

**Comment by Rishikesh6521** [comment]
Original Bug ID: #7418
